sopel/bot.py

```diff
@@ -110,6 +116,7 @@ def __init__(self, config, daemon=False):
         :class:`sopel.tools.target.Channel` objects which contain the users in
         the channel and their permissions.
         """
+
```

Comment:
Sigh, more out-of-reach lines.

Both above and below here, there are mentions of `Identifiers` that could be linked to <tt>:class:\`sopel.tools.Identifier\`</tt>

Below, there are also two docstrings (for `memory` and `shutdown_methods`) that should have trailing `.` added.